lib/fw.c

```diff
@@ -773,6 +783,12 @@ static int switchtec_fw_info_metadata_gen4(struct switchtec_dev *dev,
 	return -1;
 }
 
+static int switchtec_fw_part_info_gen4(struct switchtec_dev *dev,
+				       struct switchtec_fw_image_info *inf)
+{
+	return switchtec_fw_info_metadata_gen4(dev, inf);
```

Comment:
Not really sure the point of this helper.... it's a wrapper that does nothing, extra noise.

Reply:
Oh never mind, I should have looked at the full series before commenting.